docs/application/dotnet/get-started/wearable/first-xaml-app.md

```diff
@@ -0,0 +1,321 @@
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+        }
+    }
+}
+```
+
+The `CirclePage` class is derived from the [ContentPage](https://docs.microsoft.com/en-us/dotnet/api/xamarin.forms.contentpage?view=xamarin-forms) class, which displays a single view.
+
+The `MainPage` class derives from `CirclePage`, but notice the `partial` class definition. When Visual Studio builds the project, it parses the XAML file to generate `TizenWearableXamlApp1.MainPage.xaml.g.cs`. This file is the other partial class definition of `MainPage` that contains the definition of the `InitializeComponent` method called from the `MainPage` constructor. These two partial `MainPage` class definitions can then be compiled together. Depending on whether the XAML is compiled or not, either the XAML file or a binary form of the XAML file is embedded in the executable.
+
+
+### Add a Button and the Second Label
+
+The basic template uses a label which displays text in an area of the screen. The properties inherited from the base classes of `Label` give control over the display: font attributes, families, and sizes, as well as layout options. Modify the application by adding a button control. It is similar to the label, but is specifically designed to react to click events. As a result, the `Button` class defines the `Clicked` event, which tells the application what to do when the click event takes place.
+
+In order to do something visible on the screen to show that you have received the click event, define another label. Give the new label a value to be displayed in the initial state, and make the button click event handler update the text and button color once the click event triggers.
+
+To modify the application by adding a button and label, edit `MainPage.xaml` and `MainPage.xaml.cs`.
+
+The following code snippet shows the changes of `MainPage.xaml`:
+```diff
+<c:CirclePage xmlns="http://xamarin.com/schemas/2014/forms"
+          xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+          xmlns:c="clr-namespace:Tizen.Wearable.CircularUI.Forms;assembly=Tizen.Wearable.CircularUI.Forms"
+          x:Class="TizenWearableXamlApp1.MainPage">
+<c:CirclePage.Content>
+ <StackLayout>
+-      <Label Text="Welcome to Xamarin.Forms!"
++      <Label Text="Welcome to Tizen!"
+       VerticalOptions="CenterAndExpand"
+       HorizontalOptions="CenterAndExpand" />
++      <Button Text="Click here!"
++          BackgroundColor="Red"
++          HorizontalOptions="CenterAndExpand"
++          VerticalOptions="CenterAndExpand"
++          Clicked="OnButtonClicked" />
++      <Label x:Name="valueLabel" Text="unclicked"
++          HorizontalOptions="CenterAndExpand"
++          VerticalOptions="CenterAndExpand" />
+ </StackLayout>
+</c:CirclePage.Content>
+</c:CirclePage>
+```
+
+The `Text` atrribute of the previous `Label`was changed. A `Button` and new `Label` were added. The following code snippet shows the changes of `MainPage.xaml`:
```

Comment:
There is typo, ".cs" was missed: MainPage.xaml.cs
You are so kind but you don't need to attach a screenshot here. Although you didn't do like this, author understand what you mean because the author write this page.